_posts/2018-03-28_ncc-media-review.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: Glacier mass loss commitment
+subtitle: New Nature Climate Change study
+author: Nicolas Champollion
+date: 2018-03-28T12:00:00
+tags: ncc paper
+---
+
+## Innsbruck \& Bremen collaboration
+
+A recent study jointly conducted by the **University of Bremen and the University of Innsbruck** has just been published in Nature Climate Change about glacier mass loss commitment. Main message can be summarized as follow: **if all greenhouse gaz emissions stop today, one third of the total glacier ice mass is commited to melt at long-term** ; if we reach a new climate equilibirum of the global mean temperature at 1.5, 2 and 3K above pre-industrial values, aroun 50, 60 and 75\% respectively are commited to melt. Those results can be translated into CO2 emissions thanks to the quasi-linear relationship between cumulative anthropogenic CO2 emissions and global mean temperature. It results that today each kg of emmited C02 into the atmosphere generates the commitment of ice melt around 15 kg at long-term. This amount of commited ice melt drops to 10, 9 and 6 kg if the global mean temperature is stabilized to 1.5, 2 and 3K respectively.
+
+<a href="/_posts/marzeion_figure1.png"><img src="/_posts/marzeion_figure1.png" alt="Image missing" width="100%" /></a>
```

Comment:
Please add the image to this PR, and put it in the ``/img/blog`` folder instead, along with the other images.

Reply:
Yes, done now ! However, I'm not allowed to create a folder with the name of the blog post to put the image inside.

Reply:
I did not understand exactly how but I put the image in the /img/blog/ in the master branch and not the second branch ...